package.bzl

```diff
@@ -31,7 +31,7 @@ def rules_typescript_dev_dependencies():
         http_archive,
         name = "build_bazel_rules_nodejs",
         sha256 = "06cb04f4f745e37d542ec6883a2896029715a591c0e44c5d250a268d3752f865",
-        urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.32.0/rules_nodejs-0.32.0.tar.gz"],
+        urls = ["https://github.com/bazelbuild/rules_nodejs/archive/0.32.0.tar.gz"],
```

Comment:
This url is not correct, there is a difference between the released package and the archived one

Reply:
Thanks, done.